docs/README.md

```diff
@@ -1,195 +1,83 @@
-
-## Contributors (
+| Prop name              | Type      | Description                                                                                              | Default value |
+| ---------------------- | --------- | -------------------------------------------------------------------------------------------------------- | ------------- |
+| `variation`            | `Enum`    | Define Minicart variation. (values: `'popup'`, `'drawer'` or `'link'`)                                   | `'sidebar'`   |
+| `openOnHover`          | `Boolean` | Whether the popup minicart should open when the user hovers over it.                                     | false         |
+| `linkVariationUrl`     | `String`  | The link associated to the `'link'` variation of the Minicart.                                           | false         |
+| `drawerSlideDirection` | `Enum`    | Controls the slide direction for the `'sidebar'` variation. (values: `'rightToLeft'` or `'leftToRight'`) | `rightToLeft` |
+| `maxDrawerWidth`       | `Number`  | The maximum width in pixels for the open sidebar Minicart.                                               | `440`         |
+
+## Customization
+
+In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).
+
+| CSS Handles                     |
+| ------------------------------- |
+| `minicartWrapperContainer`      |
+| `minicartContainer`             |
+| `minicartContentContainer`      |
+| `minicartProductListContainer`  |
+| `minicartTitle`                 |
+| `minicartFooter`                |
+| `minicartSideBarContentWrapper` |
+| `minicartIconContainer`         |
+| `minicartQuantityBadge`         |
+| `popupWrapper`                  |
+| `popupContentContainer`         |
+| `arrowUp`                       |
+| `popupChildrenContainer`        |
+
+## Contributors
 
 Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
 
 <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
 <!-- prettier-ignore -->
 <table>
   <tr>
-    <td align="center"><a href="https://github.com/lucasayb"><img src="https://avatars2.githubusercontent.com/u/17356081?v=4" width="100px;" alt="Lucas Ant�nio Yamamoto Borges"/><br /><sub><b>Lucas Ant�nio Yamamoto Borges</b></sub></a><br /><a href="https://github.com/vtex-apps/minicart/commits?author=lucasayb" title="Code">=�</a></td>
+    <td align="center"><a href="https://github.com/lucasayb"><img src="https://avatars2.githubusercontent.com/u/17356081?v=4" width="100px;" alt="Lucas Ant�nio Yamamoto Borges"/><br /><sub><b>Lucas Ant�nio Yamamoto Borges</b></sub></a><br /><a href="https://github.com/vtex-apps/minicart/commits?author=lucasayb" title="Code">=�</a></td>
```

Comment:
What happened here?

Reply:
Uuhm... I did not notice this

Reply:
Fixed this in the PR regarding documentation. I'll rebase this and it should be fixed :)